xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-ui/src/main/java/org/xwiki/test/ui/po/BasePage.java

```diff
@@ -200,7 +200,7 @@ protected void clickEditSubMenuEntry(String id)
     public void edit()
     {
         WebElement editMenuButton =
-            getDriver().findElement(By.xpath("//div[@id='tmEdit']/a[contains(@role, 'button')]"));
+            getDriver().findElement(By.xpath("//div[@id='tmEdit']//a"));
```

Comment:
Why did you add this double slash? The location of the link shouldn't have changed.